From 1e56c6abb1901aa482936d8d809ebd489d2a48dd Mon Sep 17 00:00:00 2001
From: student <student@ReliableEmbeddedSystems.com>
Date: Wed, 30 May 2018 20:04:18 +0200
Subject: [PATCH] revert patch e7e73b10d690c5352cb11b1941a09e4f3dc4c8ce because
 I can not use imx6q-phytec-mira-rdk-nand with it - no network

Signed-off-by: student <student@ReliableEmbeddedSystems.com>
---
 drivers/net/ethernet/freescale/fec_main.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/freescale/fec_main.c b/drivers/net/ethernet/freescale/fec_main.c
index eb2ea23..aca7e10 100644
--- a/drivers/net/ethernet/freescale/fec_main.c
+++ b/drivers/net/ethernet/freescale/fec_main.c
@@ -3458,10 +3458,12 @@ fec_probe(struct platform_device *pdev)
 			goto failed_regulator;
 		}
 	} else {
+        #if 0
 		if (PTR_ERR(fep->reg_phy) == -EPROBE_DEFER) {
 			ret = -EPROBE_DEFER;
 			goto failed_regulator;
 		}
+        #endif
 		fep->reg_phy = NULL;
 	}
 
-- 
2.7.4

